src/services/rfqm_service.ts

```diff
@@ -60,6 +62,12 @@ export interface MetaTransactionRfqmQuoteResponse extends BaseRfqmQuoteResponse
 
 export type FetchFirmQuoteResponse = MetaTransactionRfqmQuoteResponse;
 
+export interface RfqmFee {
```

Comment:
Can you import this from `quote-server` package?

Reply:
👍   I had a reason for doing this, but after thinking about it, that reason isn't very good. Will do